diagnostic.md

```diff
@@ -32,14 +36,45 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
```

Comment:
That's a jquery-ajex request, a setter/getter example would be .val, and .val(input value)